salt/kubernetes-minion/kubelet.jinja

```diff
@@ -26,8 +26,8 @@ KUBELET_ARGS="\
 {% endif -%}
     --node-ip={{ grains['ip4_interfaces']['eth0'][0] }} \
     --config=/etc/kubernetes/manifests \
-{% if salt['pillar.get']('pod_infra_container_image') -%}
-    --pod-infra-container-image={{ pillar['pod_infra_container_image'] }} \
+{% if grains['lsb_distrib_id'] == "CAASP" -%}
+    --pod-infra-container-image=sles12/pause:1.0.0 \
```

Comment:
To do the `grains['lsb_distrib_id'] == "CAASP"` check here is good, but why don't we keep the image name in a pillar variable?